automation/01-install-rucio.bash

```diff
@@ -0,0 +1,285 @@
+fi
+if [[ "${WILL_STOP_PODS,,}" == "y" ]]; then
+  while true; do
+    echo ""
+    echo "⤑ It will stop all the pods. It could take several seconds or minutes."
+    helm uninstall daemons --debug 2>/dev/null || true
+    helm uninstall server --debug 2>/dev/null || true
+    helm uninstall postgres --debug 2>/dev/null || true
+    kubectl delete job daemons-renew-fts-proxy-on-helm-install 2>/dev/null || true
+    kubectl delete pvc data-postgres-postgresql-0 2>/dev/null || true
+    kubectl delete -f ../fts.yaml --all=true --recursive=true
+    kubectl delete -f ../ftsdb.yaml --all=true --recursive=true
+    kubectl delete -f ../xrd.yaml --all=true --recursive=true
+    kubectl delete -f ../client.yaml --all=true --recursive=true
+    kubectl delete -f ../init-pod.yaml --all=true --recursive=true
+    kubectl get all
+    if [[ "$(kubectl get all -o custom-columns=NAME:metadata.name --no-headers | wc -l)" -le 1 ]]; then
+      break
+    fi
+    sleep 4
+  done
+fi
+
+echo ""
+echo "# --------------------------------------"
+echo "# Start local environment"
+echo "# --------------------------------------"
+
+echo "┌──────────────────────────┐"
+echo "⟾ Add repositories to helm │"
+echo "└──────────────────────────┘"
+helm repo add stable https://charts.helm.sh/stable
+helm repo add bitnami https://charts.bitnami.com/bitnami
+helm repo add rucio https://rucio.github.io/helm-charts
+helm repo update
+
+echo "┌────────────────┐"
+echo "⟾ Start minikube │"
+echo "└────────────────┘"
+minikube status | grep "Stopped" &>/dev/null && {
+  echo "${MINIKUBE_HAS_KUBERNETES}"
+  minikube status || true
+  minikube start --memory="${MINIKUBE_MEMORY_RAM}" --cpus="${MINIKUBE_CPU}"
+}
+
+echo "┌────────────────────────┐"
+echo "⟾ Helm: Install Postgres │"
+echo "└────────────────────────┘"
+KUBECTL_HAS_PVC="The kubectl has the Postgres PVC. It will delete."
+kubectl get pvc data-postgres-postgresql-0 &>/dev/null || false && {
+  echo "${KUBECTL_HAS_PVC}"
+  kubectl delete pvc data-postgres-postgresql-0
+}
+helm delete postgres 2>/dev/null || true
+helm install postgres bitnami/postgresql -f ../postgres_values.yaml
+
+echo "┌──────────────────────────┐"
+echo "⟾ kubectl: Postgres set up │"
+echo "└──────────────────────────┘"
+echo "⤑ It will wait until the Postgres is set up. It could take several seconds or minutes."
+while [[ "$(kubectl get pods postgres-postgresql-0 -o custom-columns=STATUS:.status.containerStatuses[*].ready --no-headers || false)" != true ]]; do
+  echo ""
+  kubectl get pods postgres-postgresql-0 || true
+  sleep 4
+done
+echo ""
+kubectl get pods postgres-postgresql-0 || true
+
+echo "┌─────────────────────────────────┐"
+echo "⟾ kubectl: Rucio - Init Container │"
+echo "└─────────────────────────────────┘"
+kubectl delete pod init 2>/dev/null || true
+kubectl apply -f ../init-pod.yaml
+echo "⤑ It will wait until the Rucio Init Container is set up. It could take several seconds or minutes."
```

Comment:
same

Reply:
Fixed in the commit 84bf9a925933eebb4b73482beb79825180033d7d.